crates/bevy_app/src/app_builder.rs

```diff
@@ -125,6 +125,33 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a system that is run for every frame
+    ///
+    /// Systems are the main building block in bevy ECS model. You can define
+    /// normal rust functions, and call `.system()` to make them be bevy systems.
+    ///
+    /// System functions can have parameters, through with one can query and
+    /// mutate bevy ECS states. See bevy book for extra information.
+    ///
+    /// Systems are run in parallel, and the execution order is not deterministic.
+    /// If you want more fine-grained control for order, see `add_system_to_stage`
+    ///
+    /// For adding a system that runs only at app startup, see `add_startup_system`
+    ///
+    /// ## Example
+    /// ```
+    /// use bevy_app::prelude::*;
+    /// use bevy_ecs::prelude::*;
```

Comment:
this line is duplicated

Reply:
.system() seems to come from bevy_ecs instead of bevy_app

Reply:
Oh right I totally misread that.